runtime/compiler/optimizer/DataAccessAccelerator.cpp

```diff
@@ -1150,7 +1150,7 @@ bool TR_DataAccessAccelerator::generateI2PD(TR::TreeTop* treeTop, TR::Node* call
 
       if (isByteBuffer)
          {
-         /* We will be creating a precision diamond for the fast / slow path and eliminating the original call.
+         /* We will be creating a precision diamon.d for the fast / slow path and eliminating the original call.
```

Comment:
why the change from diamond to diamon.d?

Reply:
That's damage... My notes says I was trying to replace `diamon.` with `diamond.` and apparently failed miserably: https://github.com/eclipse/openj9/commit/04f3beec2f0d7bc0d29a58232a6bbf8899a9c276#diff-c5d96ecb7f8a1e47f49bf4df0c5f880dR190